assets/scss/modules/_box.scss

```diff
@@ -0,0 +1,10 @@
+@mixin dp-box ($color, $width){
+  border-radius: $dp-border-radius;
+  width: $width;
+  max-width: 408px;
```

Comment:
Can we dynamically calculate this value based on how many columns the row of boxes should have? That would be really useful to have in this mixin.

Reply:
ok I modify it